src/root-config-guide/root-config-guide.component.js

```diff
@@ -0,0 +1,192 @@
+          <a href="https://github.com/systemjs/systemjs" target="_blank">
+            systemjs
+          </a>{" "}
+          is a polyfill for this specification, which lets us use import maps in
+          browsers that don't yet support them. We will use an import map to
+          control where to download your applications from, which sets you up
+          for easy independent deployability once you get things working
+          locally.
+        </p>
+        <p>
+          If you've been following this guide, and created a single-spa react
+          application using our CLI (create-single-spa), how about checking the{" "}
+          <a
+            href={
+              window.importMapOverrides.getOverrideMap().imports[
+                application.name
+              ]
+            }
+            target="_blank"
+          >
+            JS generated by webpack now?
+          </a>{" "}
+          What is important for now, is this line:
+        </p>
+        <Code code={`System.register(["react","react-dom"], function(...`} />
+        <p>
+          This is how systemjs manages dependencies for systemjs modules. So,
+          before running your single-spa react application, systemjs need to
+          know what to do with "react" and "react-dom"! Just add them to your
+          root-config import map and you are ready to go! Also, don't forget to
+          add both the root-config and single-spa application!
+        </p>
+        <Code code={code} />
+        <div className="copy-row">
+          <button
+            type="button"
+            className="primary"
+            onClick={() => {
+              setCopying(true);
+              copy(code);
+            }}
+            disabled={copying}
+          >
+            Copy import map
+          </button>
+        </div>
+        <p>
+          But how does my application runs in the playground? Well, for your
+          convenience, we already added "react" and "react-dom" in our import
+          map! Just inspect the DOM and see for yourself!
+        </p>
+
+        <p>Now we need to start the root-config. Run:</p>
+        <Code code={`yarn start`} />
+        <p>
+          But wait... Why am I only seeing a blank page? Well, did you
+          registered you single-spa application? You can add them to your import
+          map, but if you never register them in single-spa{" "}
+          <a href="https://single-spa.js.org/docs/api.html#registerapplication">
+            registerApplication
+          </a>
+          , it will never bootstrap your applications! Just add this code in you
+          root-config entrypoint file.
+        </p>
+        <Code code={registerAppCode} />
+        <p>
+          And that's all for now! Feedbacks would be appreciated,{" "}
+          <SlackLink>join us in our slack</SlackLink> and give us a hello!
+        </p>
+      </article>
+    </>
+  );
+}
+
+function copy(str) {
+  const el = document.createElement("textarea");
```

Comment:
small trick I learned recently:

```js
const el = Object.assign(document.createElement('textarea'), {
  value: str,
  readonly: true,
})
```

Unfortunately you can't do this with style.position or style.left - the style property on dom elements doesn't work with Object.assign. But you can for other properties.

Reply:
Nice!